CMakeLists.txt

```diff
@@ -19,7 +19,11 @@
 cmake_minimum_required(VERSION 3.15)
 cmake_policy(SET CMP0074 NEW)
 
-project(Makou_Reactor LANGUAGES CXX)
+set(RELEASE_NAME "Makou_Reactor")
```

Comment:
the project call sets your Name and Release Version 

`project(makoureactor VERSION 1.8.5 DESCRIPTION Edit all your FF7 Fields LANGUAGES CXX)`